build-tools/rpm/macros.yast

```diff
@@ -15,49 +15,85 @@
 %yast_controldir %{yast_dir}/control
 %yast_schemadir %{yast_dir}/schema
 %yast_scrconfdir %{yast_dir}/scrconf
-%yast_desktopdir %{_prefix}/share/applications/YaST2
+%yast_desktopdir %{_datadir}/applications/YaST2
 
 %yast_ybindir %{_prefix}/lib/YaST2/bin
 %yast_ystartupdir %{_prefix}/lib/YaST2
-%yast_plugindir %{_libdir}/YaST2/plugin
-%yast_includedir %{_prefix}/include/YaST2
-%yast_docdir %{_prefix}/share/doc/packages/%{name}
-%yast_logdir /var/log/YaST2
-%yast_vardir /var/lib/YaST2
-%yast_fillupdir /var/adm/fillup-templates
+%yast_libdir %{_libdir}/YaST2
+%yast_plugindir %{yast_libdir}/plugin
+%yast_includedir %{_includedir}/YaST2
+%yast_docdir %{_docdir}/%{name}
+%yast_logdir %{_localstatedir}/log/YaST2
+%yast_vardir %{_sharedstatedir}/YaST2
+%yast_fillupdir %{_localstatedir}/adm/fillup-templates
 
 %yast_execcompdir %{_prefix}/lib/YaST2
 %yast_agentdir %{yast_execcompdir}/servers_non_y2
 
-# build the yast module using autotools/make
-%yast_build \
-    %{_prefix}/bin/y2tool y2autoconf \
-    %{_prefix}/bin/y2tool y2automake \
-    autoreconf --force --install \
+# prepare sources for build
+%yast_prep \
+    %{_bindir}/y2tool y2autoconf \
+    %{_bindir}/y2tool y2automake \
+    \
+    # '-Wobsolete' exposes deprecated / obsolete Autotools-macros \
```

Comment:
I can change it to `-Wall`, if you'd prefer…  My intention behind `-Wobsolete` was:  Getting information about obsoleted / deprecated Autotools-macros, but not having possible clutter about e.g. portability-stuff…